keywords.md

```diff
@@ -0,0 +1,680 @@
+---
```

Comment:
You should rename `keywords-and-reserved-words.md` to `keywords.md` instead of creating a new file `keywords.md`. See https://github.com/pingcap/docs-cn/pull/3246/files for the exact changes you should make. Also you need to update TOC.md as https://github.com/pingcap/docs-cn/pull/3246/files. @SE-Bin

Reply:
Please sign the CLA [here](https://cla-assistant.io/pingcap/docs?pullRequest=3063), thanks~ @SE-Bin